commons-numbers-fraction/src/test/java/org/apache/commons/numbers/fraction/BigFractionTest.java

```diff
@@ -625,10 +625,10 @@ void testSubtract() {
     @Test
     void testEqualsAndHashCode() {
         final BigFraction zero = BigFraction.of(0, 1);
-        Assertions.assertTrue(zero.equals(zero));
-        Assertions.assertFalse(zero.equals(null));
-        Assertions.assertFalse(zero.equals(new Object()));
-        Assertions.assertFalse(zero.equals(Double.valueOf(0)));
+        Assertions.assertEquals(zero, zero);
+        Assertions.assertNotEquals(null, zero);
+        Assertions.assertNotEquals(zero, new Object());
+        Assertions.assertNotEquals(zero, Double.valueOf(0), 0.0);
```

Comment:
Remove the delta tolerance here. And two lines above please reverse the order of null and zero.